cpp/memilio/epidemiology/state_age_function.h

```diff
@@ -811,6 +812,10 @@ struct ErlangDensity : public StateAgeFunction {
  */
 struct StateAgeFunctionWrapper {
 
+    StateAgeFunctionWrapper()
+        : m_function(mio::ExponentialSurvivalFunction(2.0).clone())
```

Comment:
I think you did that to be able to use SAFWrapper in CustomIndexArray, right? I still need to think about this.

Reply:
I think that's fine :) However, I would suggest to use a SmootherCosine as default StateAgeFunction with a low support_max to not have extensive calculations if the aprameters were not set correctly.
```suggestion
        : m_function(mio::SmootherCosine(1.0).clone())
```